elixir/apps/api/lib/api/endpoint.ex

```diff
@@ -24,9 +24,9 @@ defmodule API.Endpoint do
   # https://hexdocs.pm/phoenix/Phoenix.Logger.html
   plug Plug.Telemetry, event_prefix: [:phoenix, :endpoint]
 
-  socket "/gateway", API.Gateway.Socket, API.Sockets.options()
-  socket "/client", API.Client.Socket, API.Sockets.options()
-  socket "/relay", API.Relay.Socket, API.Sockets.options()
+  socket "/gateway", API.Gateway.Socket, API.Sockets.options(timeout: :timer.seconds(37))
+  socket "/client", API.Client.Socket, API.Sockets.options(timeout: :timer.seconds(307))
+  socket "/relay", API.Relay.Socket, API.Sockets.options(timeout: :timer.seconds(41))
```

Comment:
We use prime numbers so that if we something time-based in logs it's easy to find out which timeout that was. Otherwise, `37` would be fine for both relays and gateways. Relays have higher timeout because we control them and they should have much higher chances to stop gracefully.